lib/kitchen/directions/bake_unnumbered_tables.rb

```diff
@@ -5,6 +5,8 @@ def self.v1(book:)
         book.tables('$.unnumbered').each do |table|
           table.wrap(%(<div class="os-table">))
           table.remove_attribute('summary')
+          table.unstyled? ? table.parent.add_class('os-unstyled-container') : 0
```

Comment:
like the one-liner but you can get by without the false `0` unused return value with

```ruby
table.parent.add_class('os-unstyled-container') if table.unstyled?
```